### PR TITLE
OD-297 [Fix] Overlay filter's height

### PR DESCRIPTION
**Pull request by ivandevupp**
@sofiiakvasnevska
## Issue
OD-297 https://weboo.atlassian.net/browse/OD-297
## Description
Filter overlay now is not hidden when the LFD is under an Accordion.
## Screenshots/screencasts
https://screencast-o-matic.com/watch/cYlbo48vWd
## Backward compatibility
This change is fully backward compatible.
## Reviewers
@upplabs-alex-levchenko

### CODE REVIEW

**Review by tonytlwu — CHANGES_REQUESTED** [review]
Lacks explanation on how this works

**Comment by ivandevupp** [comment]
When LFD is added in Accordion it narrows filter in overlay because it has auto height, now its height is set to viewport height with padding bottom of  `safe-area-inset-top` to show all items when they are expanded. These changes affect only on overlay filter.

File: css/layout-css/small-card-style.upload.css [hunk 1]
```diff
@@ -18,6 +18,8 @@
   background-color: transparent;
   transform: translate3d(100%, 0, 0);
   transition: all 0.2s ease-out;
+  height: 100vh;
```
Comment (tonytlwu) [inline css/layout-css/small-card-style.upload.css:21]: If the height is now statically set to `100vh` with a padding at the bottom, then does this mean `bottom: 0` is no longer necessary? If so, please remove it.

Comment (ivandevupp) [reply css/layout-css/small-card-style.upload.css:20]: Yes, thank you, I removed it.

**Review by tonytlwu — APPROVED** [review]
@sofiiakvasnevska This is ready for testing

Risk: Medium
Affected: LFD users with filter overlays on
Verify: Filter overlays look correct

Squash and merge: Set a height for filter overlays

**Pull request merged**